ci-operator/jobs/openshift/kubernetes/openshift-kubernetes-master-presubmits.yaml

```diff
@@ -1633,6 +1633,74 @@ presubmits:
         secret:
           secretName: result-aggregator
     trigger: (?m)^/test( | .* )e2e-openstack-csi-cinder,?($|\s.*)
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - ^master$
+    - ^master-
+    cluster: build02
+    context: ci/prow/e2e-openstack-csi-manila
+    decorate: true
+    decoration_config:
+      skip_cloning: true
+    labels:
+      ci-operator.openshift.io/prowgen-controlled: "true"
+      pj-rehearse.openshift.io/can-be-rehearsed: "true"
+    name: pull-ci-openshift-kubernetes-master-e2e-openstack-csi-manila
```

Comment:
Suggest setting `optional: true` so that the job doesn't block merge once triggered.

Reply:
@marun done